alphabet/alphabet_contract.go

```diff
@@ -31,7 +31,7 @@ func init() {
 // OnNEP17Payment is a callback for NEP-17 compatible native GAS and NEO contracts.
 func OnNEP17Payment(from interop.Hash160, amount int, data interface{}) {
 	caller := runtime.GetCallingScriptHash()
-	if !common.BytesEqual(caller, []byte(gas.Hash)) && !common.BytesEqual(caller, []byte(neo.Hash)) {
+	if !common.BytesEqual(caller, interop.Hash160(gas.Hash)) && !common.BytesEqual(caller, interop.Hash160(neo.Hash)) {
```

Comment:
Do we really need to cast to `interop.Hash160`? `common.BytesEqual` accepts `[]byte`

Reply:
`gas.Hash` and `neo.Hash` are string constants, that's why we have to cast it either to `[]byte` or `interop.Hash160` in this case, since it is just an alias.

Reply:
so would it be better for readability to revert back or to keep changes?

Reply:
`interop.Hash160` type does not provide API to cast it to `[]byte`, so I'd recommend to keep `string -> []byte` cast.

Reply:
ok, rolled back